std_msgs/Header header
uint16 rotor1_speed
uint16 rotor2_speed
uint16 rotor3_speed
uint16 rotor4_speed